test/events/topic_event.js

```diff
@@ -379,7 +403,7 @@ contract('TopicEvent', function(accounts) {
             assert.isBelow(await getBlockNumber(), testTopicParams._bettingEndBlock); 
 
             try {
-                await centralizedOracle.bet(3, { from: better1, value: 1 });
+                await centralizedOracle.bet(4, { from: better1, value: 1 });
                 assert.fail();
```

Comment:
Do you need a constant for 4 as well?

Reply:
or still value = 3, and use var + 1 in tests.

Reply:
good call, added a const for invalid result index